docs/docsource/operation/Orbit_UI/Location_Management/index.rst

```diff
@@ -95,6 +95,12 @@ file to find the relevant export path, hostname, and NFS options. Use
 ``nfsstat`` on the NFS host to discover the relevant NFS version and
 protocol.
 
+.. caution::
+
+   Do not configure CRR for NFS mounts unless there is a routine
+   downscan
+   but before replication is complete are not replicated.
```

Comment:
this paragraph is a little confusing

Reply:
Ahem... 

Errm... lemme take another look at that.

Reply:
@bert-e wait

Reply:
fixed.